vcluster/configure/vcluster-yaml/control-plane/components/coredns.mdx

```diff
@@ -9,43 +9,176 @@ description: Configure CoreDNS.
+
+### Deploy separate CoreDNS
+
+A standard virtual cluster deployment consists of two pods per virtual cluster instance: the virtual cluster pod containing the API server and syncer components, and a separate CoreDNS pod. Each virtual cluster operates its own DNS service so that pods and services can locate each other by hostnames, with the syncer ensuring that standard Kubernetes DNS naming logic applies.
+
+The separate deployment model isolates DNS operations from the API server and sync functions. CoreDNS listens on port 1053 instead of port 53 to avoid conflicts with the host cluster's DNS service. DNS issues can be debugged, updated, or restarted independently without affecting the virtual cluster's control plane availability. However, virtual cluster pods cannot directly access host cluster services via DNS due to isolation boundaries.
+
+### Deploy embedded CoreDNS
+
 <ProAdmonition/>
 
-## Separate CoreDNS
+The embedded CoreDNS feature allows you to run CoreDNS as part of the syncer, which saves the overhead of an external CoreDNS pod. This enterprise feature consolidates all virtual cluster components into a single pod, reducing pod count and optimizing resource allocation.
+
+The embedded approach requires consideration of operational trade-offs. Troubleshooting might become more complex since DNS and sync functions share the same container and process space. Updates and restarts affect both DNS and sync capabilities simultaneously, though all CoreDNS features remain supported.
+
+## CoreDNS uses port 1053
+
+CoreDNS in virtual clusters listens on port 1053 instead of the default DNS port 53 to avoid conflicts with the host cluster's DNS.
+
+The host cluster already operates its own DNS service on the standard port 53, so the virtual cluster DNS services cannot bind to the same port. Since multiple virtual clusters might run on the same host cluster, each requires its own DNS service that operates independently.
+
+Port 53 is classified as a privileged port on most operating systems and requires elevated permissions to bind to it. By using port 1053, CoreDNS containers can operate with standard user privileges.
+
+### Consider cloud provider networking
+
+Different cloud providers have varying network policies that can affect DNS traffic on port 1053.
+
+EKS clusters might require security group configuration to allow inbound traffic on port 1053 between worker nodes.
+
+GKE and AKS typically allow internal cluster traffic by default, but custom network policies may need to explicitly permit UDP traffic on port 1053 between cluster nodes.
+
+## Recognize DNS isolation and limitations
+
+Virtual cluster DNS isolation maintains clear boundaries between virtual clusters while creating specific challenges for cross-cluster communication.
+
+### DNS boundaries
+
+Virtual clusters cannot directly access host services via DNS, and host cluster pods cannot access virtual cluster services through the virtual cluster's DNS.
```

Comment:
`host cluster pods cannot access virtual cluster services through the virtual cluster's DNS`
this is not clear to me. What exactly is meant by the `through the virtual cluster's DNS`?